src/hotspot/share/gc/shenandoah/shenandoahMark.inline.hpp

```diff
@@ -69,6 +70,12 @@ void ShenandoahMark::do_task(ShenandoahObjToScanQueue* q, T* cl, ShenandoahLiveD
   if (task->is_not_chunked()) {
     if (obj->is_instance()) {
       // Case 1: Normal oop, process as usual.
+      if (ContinuationGCSupport::relativize_stack_chunk(obj)) {
```

Comment:
Ooof, this is very hot code path. Is there any way to do this without injecting the code here?

Reply:
Um, this conversation seems to be "resolved" without the answer.

Reply:
Hmm... somehow, my comments gone missing.

Yes, I am award of it, but could not find a better place, any suggestions are welcome.

ZGC takes the similar hit [here](https://github.com/openjdk/jdk/blob/master/src/hotspot/share/gc/z/zMark.cpp#L289).

Let's get this in for correctness and work on optimizing it in followup CR.

Thanks.

Reply:
Thanks for your insights, @stefank